src/_data/whats-new.yml

```diff
@@ -4,8 +4,179 @@ description: This page contains recent changes that we think you'd like to know
+  versions: 2.x
+  type: Technical
+  date: October 23, 2020
+  link: https://github.com/magento/devdocs/pull/8115
+  contributor: meker12
+  profile: https://github.com/meker12
+- description: Updated the [`web` property used in the `magento.app.yaml` properties
+    topic](https://devdocs.magento.com/cloud/project/magento-app-properties.html#web)
+    in the Cloud guide.
+  versions: 2.x
+  type: Technical
+  date: October 22, 2020
+  link: https://github.com/magento/devdocs/pull/8068
+  contributor: hguthrie
+  profile: https://github.com/hguthrie
+- description: Added Magento Community links and resources for getting support and
+    discussing issues related to Magento Cloud Docker. See [Getting support for Magento
+    Cloud Docker](https://devdocs.magento.com/cloud/docker/docker-development.html#getting-support-for-magento-cloud-docker).
+  versions: 2.x
+  type: Technical
+  date: October 20, 2020
+  link: https://github.com/magento/devdocs/pull/8077
+  contributor: meker12
+  profile: https://github.com/meker12
+- description: Added a new section to the [Redis installation guide](https://devdocs.magento.com/guides/v2.4/config-guide/redis/config-redis.html)
+    about how to configure Redis to work with Magento.
+  versions: ''
+  type: Major Update
+  date: October 20, 2020
+  link: https://github.com/magento/devdocs/pull/8073
+  contributor: jonashrem
+  profile: https://github.com/jonashrem
+- description: Added a Security section to the [Generate a local REST reference](https://devdocs.magento.com/guides/v2.4/rest/generate-local.html)
+    topic.
+  versions: 2.3.x, 2.4.x
+  type: Major Update
+  date: October 16, 2020
+  link: https://github.com/magento/devdocs/pull/8064
+  contributor: dobooth
+  profile: https://github.com/dobooth
+- description: Updated the [Module Reference Guide](https://devdocs.magento.com/guides/v2.3/mrg/intro.html).
+  versions: 2.3.6
+  type: Major Update
+  date: October 15, 2020
+  link: https://github.com/magento/devdocs/pull/8057
+  contributor: dshevtsov
+  profile: https://github.com/dshevtsov
+- description: Added Magento Cloud Patches v1.0.8 release notes.
+  versions: 2.x
+  type: Technical
+  date: October 15, 2020
+  link: https://github.com/magento/devdocs/pull/8048
+  contributor: meker12
+  profile: https://github.com/meker12
+- description: Added a configuration hierarchy table to the [Configuration management
+    for store settings](https://devdocs.magento.com/cloud/live/sens-data-over.html#configuration-logic-flow)
+    topic in the _Cloud Guide_ that describes how Magento applies configuration settings
+    that have been defined in multiple contexts.
+  versions: 2.x
+  type: Major Update
+  date: October 15, 2020
+  link: https://github.com/magento/devdocs/pull/8031
+  contributor: BarnyShergold
+  profile: https://github.com/BarnyShergold
+- description: Published all documentation updates related to the Q4 2020 Commerce
```

Comment:
Link?

Reply:
There's 125 files here, so that's why there are no links.